python-checks/src/main/java/org/sonar/python/checks/cdk/CdkUtils.java

```diff
@@ -206,5 +255,9 @@ private ResolvedKeyValuePair(ExpressionFlow key, ExpressionFlow value) {
     static ResolvedKeyValuePair build(SubscriptionContext ctx, KeyValuePair pair) {
       return new ResolvedKeyValuePair(ExpressionFlow.build(ctx, pair.key()), ExpressionFlow.build(ctx, pair.value()));
     }
+
+    public ExpressionFlow value() {
```

Comment:
I understand the reason behind this method (to use `.map(ResolvedKeyValuePair::value)`) but `key` and `value` are package private and `.value` is used in many places. I think there is a god practice to use always either field access or getter/method access (for consistency in the code), but never both at the same time. WDYT?

Reply:
I agree. Let's keep the field access for now